shared/extension/behaviour/Script.hpp

```diff
@@ -98,38 +107,40 @@ namespace extension::behaviour {
             }
         }
 
-        /// @brief Creates sequences of servos from the script file given, which is emitted as a Task.
+        /// @brief Creates sequences of servos from the script files given, which is emitted as a Task.
         /// @tparam DataType The message type containing a sequence of servo positions.
         /// @param powerplant The main NUClear powerplant
         /// @param msg DataType message which will be filled with sequences of servos and emitted as a Task.
-        /// @param script Filename of the script
-        /// @param start When the script should start executing. Default is now.
-        /// @param duration_modifier Modifies the speed the script is executed at, as a multiplier of each Frame's
-        /// duration.
+        /// @param scripts Sequence of scripts to run
+        /// @param start When the first script should start executing. Default is now.
         static void emit(NUClear::PowerPlant& powerplant,
                          std::shared_ptr<Sequence> msg,
-                         const std::string& script,
-                         const NUClear::clock::time_point& start = NUClear::clock::now(),
-                         const float& duration_modifier          = 1.0) {
-            // Load the script to a vector of Frame structs
-            auto frames = load(script).as<std::vector<Frame>>();
-            // time will be incremented through each frame
+                         std::vector<ScriptRequest> scripts,
```

Comment:
Will we ever want to run a single script, if so perhaps overload this function to one that takes a single script request.